packages/icons/package.json

```diff
@@ -1,53 +1,74 @@
 {
   "name": "@naturacosmeticos/natds-icons",
+  "displayName": "NatDS Icons",
   "version": "0.6.17",
-  "description": "Natura Design System - A collection of components used by Natura Cosmeticos",
-  "main": "dist/index.js",
-  "typings": "dist/index.d.ts",
+  "private": false,
+  "description": "A collection of icons for Natura Design System",
   "keywords": [
-    "design-system"
+    "design-system",
+    "icons",
+    "natds",
+    "natura"
   ],
+  "homepage": "https://github.com/natura-cosmeticos/natds-js/tree/master/packages/icons/README.md",
+  "bugs": {
+    "url": "https://github.com/natura-cosmeticos/natds-js/issues"
+  },
+  "license": "ISC",
   "author": "Natura Cosméticos <designsystem@natura.net>",
   "contributors": [
-    "Vicente Lyrio <vicente.lyrio@vizir.com.br>",
-    "Vagner Saraiva <vagner@vizir.com.br>",
-    "Matheus Robert Lichtnow <matheus@vizir.com.br>",
     "Daniel Castro <daniel.castro@vizir.com.br>",
+    "Jimmy Andrade <jimmy.andrade@thoughtworks.com>",
+    "Matheus Robert Lichtnow <matheus@vizir.com.br>",
+    "Vagner Saraiva <vagner@vizir.com.br>",
+    "Vicente Lyrio <vicente.lyrio@vizir.com.br>",
     "Vinícius Ponciano <vinicius.ponciano@thoughtworks.com>"
   ],
-  "license": "ISC",
-  "homepage": "https://github.com/natura-cosmeticos/natds/packages/icons",
+  "files": [
+    "dist"
+  ],
+  "main": "dist/index.js",
```

Comment:
We must also add a browser specification here.

Reply:
Well remembered! I just added the `browser` entry, as well as `jsdelivr`.